WordPress/Classes/Services/MediaImageService.swift

```diff
@@ -2,24 +2,39 @@ import UIKit
 import CoreData
 
 /// A service for retrieval and caching of thumbnails for Media objects.
-final class MediaImageService: NSObject {
+actor MediaImageService {
```

Comment:
Out of curiosity, in this particular class, what makes it better to declare it as actor rather than a plain old class?